src/components/TasksTable/index.jsx

```diff
@@ -76,7 +79,13 @@ export default class TasksTable extends Component {
         <DataTable
           items={data}
           renderRow={item => (
-            <TableRow hover tabIndex={-1} key={`${item.summary}`}>
+            <TableRow
+              hover
+              tabIndex={-1}
+              key={`${item.summary}`}
```

Comment:
Template strings are unnecessary when the content of the interpolation is just a single string:

```jsx
key={item.summary}
```